README.md

```diff
@@ -124,10 +124,16 @@ requests in the console running the HTTP server.
 
 Summarizing release manager steps:
 
-1. Update versions as necessary
-2. Do a test build following steps below
-3. Make any changes as necessary and create a PR into the repository to be packaged with the modifications from steps 1-3
+1. Update versions as necessary in the project's repository, and open a pull request
+2. Do a test build following steps in "Build a package" section below
+3. Create a PR to this repository with updated build logic (if necessary) and updated debian changelog (using `./scripts/update-changelog`). Note around the time this PR is merged, there should be a corresponding tag in the associated package code's repository. Otherwise, nightly builds will fail
```

Comment:
as mentioned in Slack, the packaging changes and source tarball can come in together in a single PR, i.e. this step is: any necessary packaging changes, updated debian changelog, signed source tarballs in `tarballs` directory

(optional: we might want to link out to the RM guide in securedrop-workstation)